tensorflow/compiler/tf2tensorrt/convert/utils.cc

```diff
@@ -117,6 +117,32 @@ string DebugString(const nvinfer1::ITensor& tensor) {
                 ", dims=", DebugString(tensor.getDimensions()), ")");
 }
 
+string DebugString(const std::vector<nvinfer1::Dims>& dimvec) {
+  string out = "[";
+  for (auto dims: dimvec) {
+    StrAppend(&out, DebugString(dims));
```

Comment:
You can also use this version of absl::StrJoin. Similar for the two routines below.
std::string StrJoin(const Range& range, absl::string_view separator,
                    Formatter&& fmt)